pkg/router/routeapihelpers/validation.go

```diff
@@ -205,6 +205,11 @@ func ExtendedValidateRoute(route *routev1.Route) field.ErrorList {
 			} else {
 				tlsConfig.CACertificate = string(data)
 			}
+			// HAProxy will fail to start if intermediate CA certs use unsupported signature algorithms.
+			// However, root CAs can still use unsupported algorithms since they are self-signed.
+			if err := validateCertSignatureAlgorithms(certs); err != nil {
+				result = append(result, field.Invalid(tlsFieldPath.Child("caCertificate"), "redacted ca certificate data", err.Error()))
```

Comment:
Does this log message identify the route, so that the invalid certificate can be found/fixed?

Reply:
This message gets written to the route's `Admitted` condition type with the reason `ExtendedValidationFailed`. It is directly written on the affected route, which makes it clear of which route is impacted.